src/SDKs/RecoveryServices.Backup/Management.RecoveryServices.Backup/Generated/BackupProtectedItemsOperations.cs

```diff
@@ -103,7 +102,7 @@ internal BackupProtectedItemsOperations(RecoveryServicesBackupClient client)
             {
                 throw new ValidationException(ValidationRules.CannotBeNull, "this.Client.SubscriptionId");
             }
-            string apiVersion = "2016-12-01";
+            string apiVersion = "2017-07-01";
```

Comment:
@frustoo2004 
Looks like an SDK for the `2016-12-01` version was never published. Which implies the next SDK to be published will be a new version of the REST API, hence major version bump

Reply:
@dsgouda , We have alerady published SDK with the following versions- "2016-12-01" and "2017-07-01". If I remember correctly, the last SDK publish date was August 2017. Let me know if you need any more information

Reply:
Adding @DheerendraRathor , who published the last SDK.

Reply:
This version corresponds to the REST API version, not the SDK version.,

Reply:
@dsgouda I don't get it. SDK was published with version 2016-12-01 in https://www.nuget.org/packages/Microsoft.Azure.Management.RecoveryServices.Backup/2.1.0-preview 

Also why major version bump is required when only API version is upgraded and no contract change has been made given that api versions are not even transparent to SDK user?

Reply:
@dsgouda , Can you please review @DheerendraRathor 's comment, and let us know of any required actions? The SDK needs to be published soon for our upcoming release, and we cannot afford to delay this.

Reply:
@shahabhijeet could you take a look

Reply:
@frustoo2004 Could you point us to the nuget package that was published with `2017-07-01` API version?
@DheerendraRathor the concern is whenever we bump the API version (`2016-12-01` to `2017-07-01` in this case) the major version number must be bumped for the package.
Hope this makes sense.

Reply:
@dsgouda This doesn't make sense to me. Why does API version change require major version bumping when everything is backward compatible? And in case of C# SDK, user can't even change api-version so SDK is pretty much a black box here. 
Last time major version bumping (from 1.3 to 2.0) was done due to real breaking changes!

Reply:
@shahabhijeet , Can you please review this?. @dsgouda , Can you please expedite the process?

Reply:
@frustoo2004 and @DheerendraRathor please understand that there are versioning requirements that .NET SDK follows.
You have couple of issues in this PR
1. Why are you still in preview when your REST spec isn't
2. Why have you bumped up your API version to begin with?
 - If you have not broken any contact, why the same changes cannot be applied to 2016-12-02
3. SDK is just a reflection of your REST spec.

4. Ideally you should be making additive change to your existing API version and not upgrade from 2016-12-01 to 2017-07-01

Since you last released, new feature has been introduced that the user will see the underlying API version on the nuget.org site (here is one such [example](https://www.nuget.org/packages/Microsoft.Azure.Management.Compute/20.0.0))

For full desktop .NET framework scenarios, slso consider how can a user who wants to use functionality introduced in 2016 API version and 2017 SxS (side by side)

Please bump up major version if not, please clarify why have you changed the underlying API version if the changes are additive and does not break contracts or is it that this new functionality will not be applied in all regions?

Reply:
@frustoo2004 @DheerendraRathor please address @shahabhijeet 's comments here. Is there a reason why the REST spec is in stable version but the Nuget package is preview?